recipes/openmm-atmmetaforce-plugin/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "openmm-atmmetaforce-plugin" %}
+{% set version = "0.2.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Gallicchio-Lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cf7975639c285ed0fab441c2ac36e3cbf6e499256a708240d4acfeb562939bae
+
+build:
+  number: 0
+  skip: true  # [not linux64]
+  skip: true  # [cuda_compiler_version in (undefined, 'None')]
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux]
+    - cmake
+
+  host:
+    - python
+    - openmm
+    - swig
```

```suggestion
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ compiler('cuda') }}  # [linux]
    - cmake
    - swig

  host:
    - python
    - openmm
```
SWIG is a build tool, so it belongs in build.